Tests/js/views/assets/ez-usermenuitemfireeventview-tests.js

```diff
@@ -84,6 +84,31 @@ YUI.add('ez-usermenuitemfireeventview-tests', function (Y) {
             container.simulateGesture('tap');
             this.wait();
         },
+
+        "Should hide the user menu": function () {
+            var container = this.view.render().get('container'),
+                that = this,
+                userMenuView = new Y.Mock(),
+                isMethodCalled = false;
+
+            Y.Mock.expect(userMenuView, {
+                method: 'hide',
+                run: function () {
+                    isMethodCalled = true;
+                }
+            });
+
+            this.view.fire('addedToUserMenu', {userMenu: userMenuView});
+
+            this.view.on('logOut', function () {
+                that.resume(function () {
+                    Y.Assert.isTrue(isMethodCalled, 'Should call the `hide` method');
```

Comment:
instead of using a boolean variable, you should use `Y.Mock.verify(userMenuView)`